pkg/collector/README.md

```diff
@@ -15,7 +15,6 @@ For further details, please refer to the specific READMEs:
 
 * [check](check/README.md)
 * [corechecks](corechecks/README.md)
-* [metadata](metadata/README.md)
```

Comment:
could you also remove the paragraph starting with `All the facilities to compute and post metadata [...]` above as well? Since it's also outdated now that the metadata package was moved out of the collector